### PR TITLE
Add new content

### PR DESCRIPTION
**Pull request by DusterTheFirst**
This pr brings a lot of changes, so I will break them down below as well as provide a rendered version of the changes at <https://dusterthefirst.github.io/bps-wiki/>. I understand that this is a big change, but I have broken it down into smaller commits so that each can be reasoned about alone.

> **Note to maintainers**\
> Since this PR contains a lot of commits, when merging, please make sure to select "Squash and Merge" and to not use the default merge functionality since it will pollute the repo's commit history.

commits 4b474270f83a0df86acd61a6329f9695f270e347...6265ba234b194288357c84ce844ef83a4f843e8f
- Introduces a jekyll dev-container which makes local development easier since all the jekyll nonsense is contained in a docker container which can be started very easily from VSCode.

commit de6bd54d0bba890e813f307df9bd499d66846ed4
- Proposes the adoption of a license, specifically [CC-BY-SA]. Adopting a license makes it easier for people to use the content from this wiki in their own works (as long as they abide by the [CC-BY-SA] license terms). It also makes it easier for and encourages people to contribute to the work.

commits b22766068f761386a22c761ab93f8569970b26ca...1dc630a3d450482071ba2754555d864540ae1ed8
- Adds a 404 page
- Adds "Edit on github" links to the bottom of each page
- Formats the existing content with prettier to make it easier to read without having to scroll sideways and therefore easier to contribute to
- Moves content from the README into the home page of the wiki
- Renames some of the articles to a standard, lower-kebab-case

commits 4b48e3ec090ff9762642e313ad197165b523ac4f...60c5ef84ec9f82f717945814b695d143c3c55529
- Bring the content in the wiki up to date with the content in the #resources discord channel

commit 0b5467791317f921daf0229f6e814adecbd967fa
- Adds a table of contents to each page for easier navigation

commits 1e72e4089f1ded9703c37b673c5c2bb62bdaaba1 and 100e0c9ff03b91c8f153601487519f9311ba9c79
- Implements changes made by @FishEatPork and @peregrine-developments in their forks respectlvly
  - These changes were not suggested directly to me, but since this is such a big change, I looked through all of the forks of this repo and check if there was any new content that should be added. These 2 forks had content that seemed like it would be useful.
    - FishEatPork's original commit: 6b5110f3619d6aefa618bc9f2f858867f6fc80ec
    - Peregrine Developments' original commit: 8f3d905afa9086625e59ba8075f219c7996bc708

commits fd8e58e972965726d9a86417aefc848fd7e64725...6c58ea1937e24058aeed78074b6d646622758fa0
- Do some housekeeping around the table of contents for each page

commits 7386fa110a29349fc900e89a2f8192d0a1d86742, 892014902dd417145cb436f07e2ad67ca4c78f59...TODO:
- Add articles created by myself about the more boring parts of rocketry, the bureaucracy/regulations
- New pages:
  - https://dusterthefirst.github.io/bps-wiki/bureaucracy/
  - https://dusterthefirst.github.io/bps-wiki/bureaucracy/launch-site.html
  - https://dusterthefirst.github.io/bps-wiki/bureaucracy/regulation.html

commit cf19cfa7660d7501e9e607f77c2c235c682a37ad
- Makes sure the first header on each page will match that in the YAML front matter so they don't go out of sync

commit a58797da6be9b98edb45041c03d16171091c7e00
- Add the about section

[CC-BY-SA]: https://creativecommons.org/licenses/by-sa/2.0/

### CODE REVIEW

**Comment by DusterTheFirst** [comment]
I have finished adding all the content that I felt needed to be in this PR

File: bureaucracy/regulation.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+[nfpa-1127]: https://www.nfpa.org/codes-and-standards/all-codes-and-standards/list-of-codes-and-standards/detail?code=1127
+[nfpa]: https://www.nfpa.org/
+[tripoli-codes]: https://www.tripoli.org/content.aspx?page_id=22&club_id=795696&module_id=494857
+[tripoli]: https://www.tripoli.org/
+
+### Netherlands
+
+-   [Regeling modelraketten][government-nl] - Netherlands national regulations regarding
+    model rocketry
+    -   > Personally, I'd highly advise going through the local amateur rocketry
```
Comment (Nufflee) [inline bureaucracy/regulation.md:81]: this quote doesn't render well, it's just a bunch of empty space
image

File: electronics/imu-selection.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+parent: Electronics
+---
+
+# _How to choose an IMU_
+
+-   Don't consider price as your main deciding factor, the cheapest option might
+    still have the worst price/performance.- Consider expected G loads, then
+    double or triple for safety margins.
```
Comment (Nufflee) [inline electronics/imu-selection.md:9]:
```suggestion
-   Don't consider price as your main deciding factor, the cheapest option might
    still have the worst price/performance.
-   Consider expected G loads, then double or triple for safety margins.
```

File: electronics/imu-selection.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+-   Note that the BNO055 has onboard sensor processing you pay extra for that
+    will NOT work in a rocket context, buy the BMX055.
+-   Spending that bit extra on a better sensor will help alleviate pain down the
+    line as you don't have to fight it as much
+-   Consider ditching breakout boards, instead design your own integration on
+    board, follow the datasheet recommended schematic and layout.
+-   Breakouts being attached to your board can lead to sensor decoupling sensed
+    motion from your airframe due to waffling around which is why we don't really
+    recommend using them even though it seems easier. SMD soldering is a useful
+    skill and opens the door to much more accurate sensor implementations.
```
Comment (Nufflee) [inline electronics/imu-selection.md:22]:
```suggestion
    -   Breakouts being attached to your board can lead to sensor decoupling sensed
        motion from your airframe due to waffling around which is why we don't really
        recommend using them even though it seems easier. SMD soldering is a useful
        skill and opens the door to much more accurate sensor implementations.
```

File: electronics/imu-selection.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Supply your sensors with a clean, quiet, and correct voltage reference for best
+results
+
+**A** = Accelerometer
+**G** = Gyroscope
+**M** = Magnetometer
+
+| Name      | Price    | [DOF]     | Ranges                   | Precision           | Drift figures                 | Library? | Recommended                                                                                    |
+| --------- | -------- | --------- | ------------------------ | ------------------- | ----------------------------- | -------- | ---------------------------------------------------------------------------------------------- |
+| MPU6050   | $2 (EOL) | 6 (A+G)   | ±16g ±2000°/sec          | 16Bit               | ±50mg(up to ±80mg) ±20º/s     | Many     | Bad drift, bad lifetime, **do not use**                                                        |
```
Comment (Nufflee) [inline electronics/imu-selection.md:38]: Doesn't have to be a part of this PR, but the "Drift figures" section is very hard to read due to all figures being on the same line and certain sensors not having all of them. 
Sources should be provided for these as well.

File: misc/books.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+parent: Miscellaneous
+---
+
+# Book recommendations
+
+<!-- Please use this article for miscellaneous book recommendations that don't specifically fit anywhere else! -->
+
+-   [Sidewinder Creative Missile Development](https://www.amazon.co.uk/Sidewinder-Creative-Missile-Development-China-ebook/dp/B00EXCOIRM/ref=sr_1_1?dchild=1&keywords=sidewinder+creative&qid=1616596630&sr=8-1)
```
Comment (Nufflee) [inline misc/books.md:9]:
```suggestion
-   [Sidewinder: Creative Missile Development at China Lake by Ron Westrum](https://www.amazon.co.uk/Sidewinder-Creative-Missile-Development-China-ebook/dp/B00EXCOIRM/ref=sr_1_1?dchild=1&keywords=sidewinder+creative&qid=1616596630&sr=8-1)
```

**Review by Nufflee — COMMENTED** [review]
LGTM other than the few suggestions I left

**Comment by DusterTheFirst** [comment]
@Nufflee The changes have already been merged by @ArsenioDev. Should I open a new PR to address the comments?

**Pull request merged**